service-worker.js

```diff
@@ -0,0 +1,69 @@
+---
+---
+// Add list of files to cache here.
+const cacheName = 'cache-v1';
+const precacheResources = [
+  '/',
+  'index.html',
+  'event-data.html',
+  'importing-exporting-data.html',
+  'season-config.html',
+  'team-data.html',
+  'styles.css',
+  'w3css.css',
+  'scripts/jquery.js',
+  'scripts/seasonConfigScript.js',
+  'scripts/eventDataScript.js',
+  'scripts/teamDataScript.js',
+  'scripts/importing-Exporting-Config.js',
+  'scripts/commonMethodHelpers.js',
+  'scripts/autosize-master/dist/autosize.js',
+  '{{site.logo}}',
+  'images/background.jpg',
+  'manifest.json',
```

Comment:
I tried to list every file here, we should add a backlog item to use Jekyll to generate this.